### DISCUSSION TITLE
Do I have to use the installer - or could I use a ready-made EFI folder? (Mojave)

### DISCUSSION THREAD

**Discussion by Bradamante3D** [osy86/HaC-Mini#960] [Q&A] [answered]
Hi there, I would like to use such an Hades Canyon NUC with Mojave 10.14.6. My problem is that I already have an existing installation, but in terms of installation methods only the installer is given here. Is this installer the only way (for example, because the Mac OS installer on the USB stick is manipulated)? I would prefer a ready-made EFI folder that I can just move over and put in the SSD (a Crucial P1 m2 SSD in this case).

Best regards and thanks

**Comment by osy** [comment, marked as answer]
The installer crafts an EFI based on the options. You can always copy it from one machine to another or from one drive to another. Note that the unique identifiers (serial) would be the same though.

**Reply by x2358473** [reply]
我试过安装Mojave 10.14.6。在这个系统 上时间会出现不走的情况。感觉最低只能Catalina .至少目前Catalina使用3.0.1的HacMini还能正常工作。 
I tried installing Mojave 10.14.6. On this system, time will not go. I feel the lowest is Catalina. At least Catalina can still work normally with 3.0.1 HacMini.
